packages/ddg2dnr/puppeteerInterface.js

```
@@ -96,39 +85,39 @@ class PuppeteerInterface {
+                return;
             }
 
             await chrome.declarativeNetRequest.updateDynamicRules({
-                removeRuleIds: Array.from(self.ruleById.keys())
-            })
+                removeRuleIds: Array.from(self.ruleById.keys()),
+            });
 
-            delete self.ruleById
-        })
+            delete self.ruleById;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)